README.md

```diff
@@ -155,6 +157,9 @@ latest version.
 
     $ make services
 
+#### Warning
+(you may have to run `sudo usermod -a -G docker $USER`, then reboot your computer to enable the current user to use docker )
+
```

Comment:
peux-tu en dire un peu plus? j'ai jamais eu à le faire sur mac, et @regisb @AlexandreCantin non plus sur U18 AFAIK

quelle erreur tu avais?

Reply:
IMHO  `s/Warning/Known issues/`

Reply:
https://techoverflow.net/2017/03/01/solving-docker-permission-denied-while-trying-to-connect-to-the-docker-daemon-socket/

Reply:
Il y a une section Knowns issues ? Ou c'est le titre de cette partie que tu veux que je change ?

Reply:
juste changer le titre (warning pour un titre c'est bizarre :) )

Reply:
ok met le lien dans le README stp

ton formating de markdown est un peu bizarre. Tu utilises un editeur de markdown avec live preview? Ce serait bien pratique. Sur MacOS j'utilise Quiver. Sur linux: https://itsfoss.com/best-markdown-editors-linux/

Reply:
Ah non mais j'ai un preview du md sur vs code

Reply:
Je refais un commit, dis moi si ça te va